core/src/test/java/nl/nn/adapterframework/xslt/XsltErrorTestBase.java

```diff
@@ -277,7 +277,7 @@ public void notifyXalanExtensionsIllegalForSaxon() throws SenderException, TimeO
 			errorMessage = e.getMessage();
 			assertThat(errorMessage,containsString("Cannot find a matching 2-argument function named {http://exslt.org/strings}tokenize()"));
 		}
-		assertThat(testAppender.getNumberOfAlerts(),is(getMultiplicity()+1+EXPECTED_NUMBER_OF_DUPLICATE_LOGGINGS));
+		assertThat(testAppender.getNumberOfAlerts(),is(1+EXPECTED_NUMBER_OF_DUPLICATE_LOGGINGS));
```

Comment:
Ik zie geen getMultiplicity() meer in het verwachte aantal alerts. Dat is goed, lijkt me: geen duplicate messages